commit/src/domain.rs

```diff
@@ -144,7 +144,7 @@ pub struct TwoAdicMultiplicativeCoset<Val: TwoAdicField> {
 
 impl<Val: TwoAdicField> TwoAdicMultiplicativeCoset<Val> {
     /// Return the element `h` which generates the subgroup `H`.
-    fn gen(&self) -> Val {
+    fn subgroup_generator(&self) -> Val {
```

Comment:
If the coset is nontrivial isn't it technically not a subgroup? We could say `coset_generator()` but I think `generator()` would also be descriptive, and then we could use the same name for the circle domain where it's not necessarily a coset.

Reply:
Ah right, `subgroup_generator` makes sense then